qiskit/user_config.py

```diff
@@ -64,16 +66,28 @@ def read_config_file(self):
                         "'latex_source'"
                         % circuit_drawer)
                 self.settings['circuit_drawer'] = circuit_drawer
-
+            # Parse circuit_mpl_style
             circuit_mpl_style = self.config_parser.get('default',
-                                                       'circuit_mpl_style')
+                                                       'circuit_mpl_style',
+                                                       fallback=None)
             if circuit_mpl_style:
                 if circuit_mpl_style not in ['default', 'bw']:
                     raise exceptions.QiskitUserConfigError(
                         "%s is not a valid mpl circuit style. Must be "
                         "either 'default' or 'bw'"
                         % circuit_mpl_style)
                 self.settings['circuit_mpl_style'] = circuit_mpl_style
+            # Parse transpile_optimization_level
+            transpile_optimization_level = self.config_parser.getint(
+                'default', 'transpile_optimization_level', fallback=-1)
```

Comment:
what does fallback=-1 do here?

Reply:
It's as opposed to None like above. I couldn't use none on getint() here so I used an invalid value, -1, to say don't change anything if transpile_optimization_level is not set in the user_config_file.